tests/environment.py

```diff
@@ -270,6 +282,15 @@ def send_bwl_event(self, event: str) -> None:
         command = "echo \"{}\" > /tmp/$USER/beerocks/{}/EVENT".format(event, self.iface_name)
         self.agent.command('sh', '-c', command)
 
+    def read_tmp_file(self, filename: str) -> bytes:
+        '''Read the file given by `filename` from the radio's status directory'''
+        path = "/tmp/$USER/beerocks/{}/{}".format(self.iface_name, filename)
+        return self.agent.command("sh", "-c", 'cat "{}"'.format(path))
```

Comment:
Why use `sh -c "cat file"` rather than simply `cat file`?
```suggestion
        return self.agent.command("cat", '"{}"'.format(path))
```

Reply:
Added comment to explain it.